brave/brave-core
# BAT Points updates for unsupported regions

Fixes brave/brave-browser#6480
Fixes  https://github.com/brave/brave-browser/issues/6577

This implements the BAT Points language for regions where only anon wallets are allowed.

<img width="429" alt="Screen Shot 2019-10-18 at 9 36 32 PM" src="https://user-images.githubusercontent.com/8732757/67137779-6544db80-f1ef-11e9-8ea2-fdc651a9dfc0.png">

## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (unit, browser, security tests) on

## Test Plan:
1. Enable rewards
2. Wait for initilization
3. Close browser
4. Open Preferences file and search for countryid_at_install
5. Replace current value with 19024
6. Restart Browser
7. Claim a grant
8. Confirm that the grant claim shows "BAT Points" instead of BAT
9. Tip a couple of sites
10. Confirm that the wallet view in `brave://rewards` adheres to spec for bat points
11. Confirm that the panel adheres to spec for bat points

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on.
